go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/urfave/cli
+
+go 1.12
+
+require (
+	github.com/BurntSushi/toml v0.3.1
+	gopkg.in/urfave/cli.v1 v1.20.0
```

Comment:
Require self? A bit strange no?

Reply:
The package `github.com/urfave/cli/altsrc`seems to rely on it :)

Reply:
I think the imports should be rewritten to use github, which I think would address this confusion?

I think gopkg is a bad idea going forward given go.mod landed?
Or am I misunderstanding stuff.

Reply:
Sure, I rewrote the imports and now the dependency is gone 👍